base/Ballerina.toml

```diff
@@ -2,7 +2,7 @@
 org = "ballerinax"
 name = "health.base"
 version = "1.0.2-snapshot"
```

Comment:
we need to bump the version as well for the release.

Reply:
This should be done in the  Prepare for release commit.. right?

Reply:
Updated